setup.py

```diff
@@ -31,6 +31,7 @@ def read(name):
 # https://github.com/nteract/papermill/issues/239
 NB = [
     "papermill",
+    "notebook<=6.5.4",
```

Comment:
```suggestion
    "notebook<7",
```

better to pin this way, because new notebook 6.x versions will come out